vllm/platforms/interface.py

```diff
@@ -145,7 +145,12 @@ def is_cuda_alike(self) -> bool:
         return self._enum in (PlatformEnum.CUDA, PlatformEnum.ROCM)
 
     def is_sleep_mode_available(self) -> bool:
-        return self._enum == PlatformEnum.CUDA
+        if self._enum == PlatformEnum.CUDA:
+            return True
+
+        from vllm.platforms.rocm import on_mi3xx
```

Comment:
Importing rocm.py directly is not safe since it tries to import several rocm libs, this should be done through another interface like current_platform